### PR TITLE
adjust login tests for mercury jenkins build

### PR DESCRIPTION
**Pull request by tomasznapieralski**
/cc @qaga

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 18 issues:
* MAJOR 2 major
* MINOR 16 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L40) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L24) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L22) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L50) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L38) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L30) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L46) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L34) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L28) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L26) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L20) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L72) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L36) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L32) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L42) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L48) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/ca6258c456694600c8bd9bfe91e78a96ac5e228e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)

**Comment by tomasznapieralski** [comment]
half of tests have failed on jenkins i will investigate it tomorrow

**Comment by WikiaTech** [comment]
SonarQube analysis reported 18 issues:
* MAJOR 2 major
* MINOR 16 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L24) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L38) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L34) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L22) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L40) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L32) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L46) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L42) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L30) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MAJOR [Remove those useless parentheses.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L72) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L48) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L50) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L26) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L36) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L20) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)
* MINOR [Move this variable to comply with Java Code Conventions.](https://github.com/Wikia/selenium-tests/blob/cd7fe572bd067f51106014417df865926cbdd1f8/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/SignupPageObject.java#L28) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java [hunk 1]
```diff
@@ -60,14 +60,16 @@ public void logUserIn(String username, String password) {
...
 
-    try{
-      redirectParameter = URLEncoder.encode(urlBuilder.getUrlForWiki(Configuration.getWikiName()), "UTF-8");
+    try {
+      redirectParameter =
+          URLEncoder.encode(urlBuilder.getUrlForWiki(Configuration.getWikiName()), "UTF-8");
 
-    } catch (UnsupportedEncodingException e){
-      PageObjectLogging.log("encoding","problem occured during URL encoding",false);
+    } catch (UnsupportedEncodingException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.mercury.TopBarComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues:
* MAJOR 9 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L126) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L117) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Move the "ellipsis" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L134) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Move the "visible" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L138) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Move the "visible" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L142) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L92) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L101) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/c93b646adb4b2c3dccabeb5be32523116d53180c/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java#L108) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java [hunk 3]
```diff
@@ -60,14 +61,16 @@ public void logUserIn(String username, String password) {
...
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java]: MAJOR Either log or rethrow this exception. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java [hunk 4]
```diff
@@ -4,8 +4,10 @@
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mercury.NavigationSideComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.mercury.TopBarComponentObject;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues:
* MAJOR 9 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L126) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L117) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Move the "visible" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L142) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L101) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L92) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MAJOR [Move the "ellipsis" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L134) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Move the "visible" string literal on the left side of this string comparison.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java#L138) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/03d7bed64c94fc9a640cd47d4871b5b508d4bad2/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java#L109) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 2 major
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Either log or rethrow this exception.](https://github.com/Wikia/selenium-tests/blob/8dd22be486c7aa119656cde98ef759bbb1480455/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/LoginPage.java#L109) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)

**Comment by tomasznapieralski** [comment]
Test are finally passing on both local and Jenkins

**Pull request merged**